public/scripts/script.js

```diff
@@ -2,8 +2,8 @@
 const menuButton = document.querySelector('header nav:nth-of-type(2) button');
 const menu = document.querySelector('header nav:nth-of-type(2) ul');
 
-menuButton.addEventListener('click', () => {
-    menu.classList.toggle('show');
+menuButton.addEventListener('click', () => { //When menu button is clicked
+    menu.classList.toggle('show'); // Toggle show class on menu
```

Comment:
Ik denk dat dit een beetje overbodig is, misschien kan je beter boven deze twee regels "toggle hamburgermenu" oid zetten :)

Reply:
Ik zie het op meerdere plaatsen, maar het geldt uiteraard op al die soortgelijke comments